package.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "daostack-alchemy",
+  "name": "daostack-vanille",
   "version": "0.0.0-alpha.3",
-  "description": "Creating DAOs for an Emergent Future",
+  "description": "Creating DAOs for a Collective Intelligence",
   "main": "dist/app.bundle.js",
   "repository": {
-    "url": "git+ssh://git@github.com/daostack/Alchemy-Aurelia.git",
+    "url": "git+ssh://git@github.com/daostack/Vanille.git",
     "type": "git"
   },
-  "author": "DAOStack",
+  "author": "DAOstack",
   "license": "CC0-1.0",
```

Comment:
> "license": "CC0-1.0", 

I though we are under GPL .

Reply:
Yes, that change is sitting in another branch waiting for this PR to be approved.  Can we just let it be there?

Reply:
Please open an issue for that so it could be tracked.

or

Change that to

>"license": "GPL-3.0", 

-copy the license file from https://github.com/daostack/Arc/blob/master/LICENSE